Formula/wxwidgets@3.0.rb

```diff
@@ -70,13 +71,13 @@ def install
     # this ensures that Python software trying to locate wxpython headers
     # using wx-config can find both wxwidgets and wxpython headers,
     # which are linked to the same place
-    inreplace "#{bin}/wx-config", prefix, HOMEBREW_PREFIX
+    inreplace bin/"wx-config", prefix, HOMEBREW_PREFIX
 
     # Move some files out of the way to prevent conflict with `wxwidgets`
-    bin.install "#{bin}/wx-config" => "wx-config-#{version.major_minor}"
+    bin.install bin/"wx-config" => "wx-config-#{version.major_minor}"
     (bin/"wxrc").unlink
     (share/"wx"/version.major_minor).install share/"aclocal", share/"bakefile"
-    Dir.glob(share/"locale/**/*.mo") { |file| add_suffix file, version.major_minor }
+    share.glob("locale/**/*.mo") { |file| add_suffix file, version.major_minor }
```

Comment:
Looking at this again, I suspect this is not necessary, since the locale files for `wxwidgets` are already suffixed.

Adding the suffix likely also breaks being able to find them, but I'm not sure.

Reply:
Went ahead and dropped it. Anyway, we only have 1 formula (`spatialite-gui`) that still uses this formula. May be worth looking into that one later given that Fedora was able to rebuild with v3.2 https://src.fedoraproject.org/rpms/spatialite-gui/blob/rawhide/f/spatialite-gui.spec#_66